### PR TITLE
Add Tesco

### PR DESCRIPTION
**Pull request by nesukun**
## Add Tesco

- [x] I have read the [contributing guidelines](../CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](../CODE_OF_CONDUCT.md)
- [x] I have followed the [format](../CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [ ] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -374,6 +374,7 @@ See [HackerNews](https://news.ycombinator.com/item?id=13874026).
 - [Tanooki Labs](http://tanookilabs.com) | New York, NY | Paid half-day take home project with followup review and discussion
 - [Tattoodo](https://www.tattoodo.com) | Copenhagen, Denmark | Takehome exercise
 - [Telus Digital](https://labs.telus.com) | Toronto & Vancouver, Canada | Discuss knowledge and interest, explore previous work, pair with developers when possible, alternatively take home project.
+- [Tesco PLC](https://www.tesco.com) | London, United Kingdom | Discussion, pair programming, pair whiteboard system design for a given scenario
```
Comment (icambridge) [inline README.md]: This uses a whiteboard?

Comment (nesukun) [inline README.md]: > Discussing a real world problem with the help of a whiteboard is 👍
> Solving basic CS questions, technical puzzles, riddles, brainteasers (with or without whiteboard) is 👎

From my experience the usage of the whiteboard in their process falls in the solving a real world problem. It is definitely not a Technical pass/fail question or a puzzle or some kind of abstract 'smart' question.

**Comment by poteto** [comment]
Can you rephrase the interview process please?

File: README.md [hunk 2]
```diff
@@ -375,6 +375,7 @@ See [HackerNews](https://news.ycombinator.com/item?id=13874026).
 - [Tattoodo](https://www.tattoodo.com) | Copenhagen, Denmark | Takehome exercise
 - [Telus Digital](https://labs.telus.com) | Toronto & Vancouver, Canada | Discuss knowledge and interest, explore previous work, pair with developers when possible, alternatively take home project.
 - [Ten Thousand Coffees](https://tenthousandcoffees.com) | Toronto, Canada | Take home project, then explain how you solved the project
+- [Tesco PLC](https://www.tesco.com) | London, United Kingdom | Discussion, pair programming, pair whiteboard system design for a given scenario
```
Comment (poteto) [inline README.md]: Can you rephrase the interview process please?

Comment (nesukun) [inline README.md]: Does `Pair programming and casual hypothetical system design discussion with the aid of a whiteboard` look good? English is not my main language so I'll gladly welcome any suggestion you may have.

Comment (poteto) [inline README.md]: I think 

> Pair programming and casual hypothetical system design discussion

would be sufficient
